packages/manager/apps/dedicated/client/app/ip/byoip/byoip.service.js

```diff
@@ -0,0 +1,58 @@
+export default class ByoipService {
+  /* @ngInject */
+  constructor($http, $q, coreConfig, User) {
+    this.$http = $http;
+    this.$q = $q;
+    this.User = User;
+    this.coreConfig = coreConfig;
+  }
+
+  getCatalog() {
+    const user = this.coreConfig.getUser();
+    return this.$http
+      .get('/order/catalog/formatted/bringYourOwnIp', {
```

Comment:
No use of `/order/catalog/public`?

Reply:
If i use `/order/catalog/public`, will have to filter on our side `/order/catalog/formatted/{catalogName}` gives a filtered specific plans. Is there any advantage/disadvantage of using these ?

Reply:
If you check the root API route here: https://api.ovh.com/console/#/order/catalog/formatted#GET, you see that this catalog is deprecated. So in fact we should avoid using it at most. That said, I don't know about the availability of byoi catalog in the public catalog

Reply:
Oh okay. I checked it and its not available in `/order/catalog/public`.
So it has to be `/order/catalog/formatted/{catalogName}`
I will check with PU as well and confirm this

Reply:
Hi @JeremyDec , It is confirmed by PU to use this API and it is actually  `/order/catalog/formatted/{catalogName} `and not https://api.ovh.com/console/#/order/catalog/formatted#GET